Makefile

```diff
@@ -22,9 +22,7 @@ ALL_SRC := $(shell find . -name "*.go" | grep -v -e vendor \
 TEST_TIMEOUT := "-timeout=3s"
 
 .PHONY: test
-test: examples
-	$(ECHO_V)go test $(RACE) $(TEST_TIMEOUT) $(PKGS)
```

Comment:
I added this because of strange behavior on race check. 

`go test $(RACE)` fails the build on race condition and `make test` returns 2. 
`$(COV_REPORT)`, on the other hand results in an error and `make test` returns 0

Any other way to fix `$(COV_REPORT)` that results in `make test` to actually error out?